friends/src/components/FriendsList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import api from "../helper/api";
+import NewFriend from "./NewFriend";
+
+function FriendsList() {
```

Comment:
In your app, if you navigate away from FriendsList and go back, does it do a axios call each time?  What can you do to combat that if it does?